portfolio/src/main/webapp/index.html

```diff
@@ -11,8 +11,14 @@
       <h1>Elizabeth Mota's Portfolio</h1>
         <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>  
-        <button onclick="getData()">Fetch a quote!</button>
-      <div id="quote-container"></div>
+    <h2>Add A Comment!</h2>
+    <form method="POST" action="/new-task">
```

Comment:
action here is still `/data`, served by the single servlet that we have. Our servlet can handle both GET and POST actions :)